### PR TITLE
Demo to Evaluate a Python file

### PR DESCRIPTION
**Pull request by ghost**
@olyagpl, please review README file.

### CODE REVIEW

File: .github/workflows/graalpy-eval-demo.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  workflow_dispatch:
+permissions:
+  contents: read
+jobs:
+  run:
+    name: Run 'graalpy-eval-demo'
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+      - uses: actions/checkout@v3
```
Comment (olyagpl) [inline .github/workflows/graalpy-eval-demo.yml:19]: Please update to `actions/checkout@v4`.

**Review by olyagpl — CHANGES_REQUESTED** [review]

File: .github/workflows/graalpy-eval-demo.yml (hunk 1 above)
Comment (bhoran) [reply .github/workflows/graalpy-eval-demo.yml]: Done

File: graalpy-eval-demo/pom.xml [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    <!-- The GraalPy dependency -->
+    <dependency>
+      <groupId>org.graalvm.polyglot</groupId>
+      <artifactId>python</artifactId>
+      <version>${graalpy.version}</version>
+      <scope>runtime</scope>
+      <type>pom</type>
+    </dependency>
+
+    <!-- The GraalPy Launcher. -->
```
Comment (timfel) [inline graalpy-eval-demo/pom.xml:79]: The launcher is not needed

File: graalpy-eval-demo/README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+# GraalPy Evaluating Demo
+This small application demonstrates how to use GraalPy to evaluate a Python file from a Java application using Maven.
```
Comment (olyagpl) [inline graalpy-eval-demo/README.md:2]: This small application demonstrates how to use GraalPy to evaluate a Python file from a Java application, built with Maven.

File: graalpy-eval-demo/README.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+The application contains a Main class that evaluates an external Python file named _Hello.py_ when run.
+
+The example also shows how to produce a native executable using [GraalVM](https://www.graalvm.org/).
+
```
Comment (olyagpl) [inline graalpy-eval-demo/README.md:6]: Double empty line

File: graalpy-eval-demo/README.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+2. Run the application:
+    ```bash
+    mvn exec:exec
+    ```  
+
+3. To confirm that your application reads the contents of _Hello.py_, edit the file, as follows:
+    ```python
+    print ("Hello Evaluated GraalPy!")
+    ```
+    then run the application again (using `mvn exec:exec`).
```
Comment (olyagpl) [inline graalpy-eval-demo/README.md:31]: then > Then

File: graalpy-eval-demo/README.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+The example also shows how to produce a native executable using [GraalVM](https://www.graalvm.org/).
+
+
+## Preparation
+
+Install a JDK 21, ensure that it is on your classpath, and set the value of `JAVA_HOME` accordingly.
+We recommend GraalVM for JDK 21 that can be easily installed using [SDKMAN!](https://sdkman.io/).
+```bash
+sdk install java 21.0.2-graal
+```
```
Comment (olyagpl) [inline graalpy-eval-demo/README.md:14]: We should probably add also:

For other download options, go to [GraalVM Downloads page](https://www.graalvm.org/downloads/).

File: graalpy-eval-demo/README.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+3. To confirm that your application reads the contents of _Hello.py_, edit the file, as follows:
+    ```python
+    print ("Hello Evaluated GraalPy!")
+    ```
+    then run the application again (using `mvn exec:exec`).
+
+
+## (Optional) Build and Run a Native Executable
+
+If you [installed GraalVM](https://www.graalvm.org/downloads/) you can build this Java-Python application into a native executable and run it.
```
Comment (olyagpl) [inline graalpy-eval-demo/README.md:36]: If you [installed GraalVM](https://www.graalvm.org/downloads/), you can ...

File: graalpy-eval-demo/README.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+2. Run the native executable:
+    ```bash
+    ./target/eval_graalpy
+    ```
+
+3. To confirm that your application reads the contents of _Hello.py_, edit the file, as follows:
+    ```python
+    print ("Hello Native Evaluated GraalPy!")
+    ```
+    then run the application again (using `./target/eval_graalpy`).
```
Comment (olyagpl) [inline graalpy-eval-demo/README.md:53]: then > Then

**Review by olyagpl — APPROVED** [review]

**Pull request merged**